helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml

```diff
@@ -14,7 +14,7 @@ generic-service:
     host: app-hostname.local    # override per environment
     tlsSecretName: hmpps-welcome-to-prison-cert
     v1_2_enabled: true
-    v0_47_enabled: true
+    v0_47_enabled: false
```

Comment:
have to set to false instead of removing it since the [default is set to true](https://github.com/ministryofjustice/hmpps-helm-charts/blob/ead8a533ef3683c8ffda83b9080723f8ed99dfe1/charts/generic-service/values.yaml#L50)